streams/src/test/java/org/apache/kafka/streams/processor/internals/metrics/StreamsMetricsImplTest.java

```diff
@@ -305,7 +297,6 @@ public void shouldGetNewTaskLevelSensor() {
             recordingLevel
         );
 
-        verify(metrics);
```

Comment:
`shouldGetNewTaskLevelSensor` invokes `setupGetNewSensorTest` which internally creates a stub with exact values, providing implicit verify.

Reply:
This is not completely true, since the sensor name is not verified. However, you are completely right with removing the verify from here.